tools/cache_perf_tool/main.go

```diff
@@ -16,53 +16,62 @@ import (
 	"io"
 	"log"
 	"os"
+	"runtime"
 	"runtime/pprof"
 	"strconv"
 	"strings"
+	"sync"
 	"sync/atomic"
 	"testing"
 	"time"
 
 	"github.com/couchbase/sync_gateway/base"
 	"github.com/couchbase/sync_gateway/db"
 	"github.com/couchbaselabs/rosmar"
+	"github.com/felixge/fgprof"
 )
 
 var numGoroutines atomic.Int32
 
+var once sync.Once
```

Comment:
Doesn't feel appropriate to be a global here with a generic name. Either make it specific to CSV header or (which I think is also possible, rework that bit of code so it doesn't need to rely on a `sync.Once`.

It seems like we can just print the CSV header before we enter the ticker loop?

Reply:
So I went with this approach as when we create the node we print logging out to show SGW nodes with their associated delay. I found that printing before loop would print the header in the middle of this logging and you'd end up with csv output and logging mixed together. I'll rename and add comments here.